.rubocop.yml

```diff
@@ -776,12 +777,21 @@ Rails/HttpPositionalArguments:
     - spec/**/*
     - test/**/*
 
+Rails/I18nLocaleAssignment:
+  Enabled: false
+
+Rails/NegateInclude:
+  Enabled: false
+
 Rails/Present:
   Enabled: false
 
 Rails/RelativeDateConstant:
   Enabled: false
 
+Rails/SquishedSQLHeredocs:
+  Enabled: false
+
 Rails/TimeZone:
```

Comment:
This would turn `<<~SQL` into `<<~SQL.squish` 

The tilde `~` instead of hypen `-` already removes leading whitespace, so the only thing `.squish` would do is remove newlines... and I think newlines can help readability of inspecting queries so I'd opt to keep them in